frontend/src/pages/HomePage.js

```diff
@@ -1,24 +1,39 @@
 import {Typography} from "@mui/material";
 import useStyles from "../styling/useStyles";
 import CardGrid from "../components/CardGrid";
+import theme from "../styling/theme";
 
 export default function HomePage({activeDepot, openPortfolioItem}) {
 
```

Comment:
```suggestion
const {name, valueOfPortfolio, arithemticalGain} = activeDepot;
```
will save you having to write activeDepot uncountable times further down and increase the readability drastically